config/searchConfig.js

```diff
@@ -0,0 +1,40 @@
+const fields = ["name", "username", "tags", "location.name"];
+const defaultPath = "name"
+function getQueryString(queryArr) {
+    console.log(queryArr);
```

Comment:
I wonder if we should let console.log statements in production. What do the maintainers think about this?

Reply:
Shall I remove the console logs? I kept them to analyze if the aggregation is working correctly or not

Reply:
Even I'm a 100 on this. Would help if a maintainer lets us know?

Reply:
Good spot, we should not have console log, we have a logger if things need to be logged

Reply:
Removed the logs
Thank you aditya for spotting it and thank you eddie for the final answer

Reply:
Great collaboration everyone ❤️ 💪